source/_components/camera.bomradarloop.markdown

```diff
@@ -0,0 +1,167 @@
+---
+layout: page
+title: "BOM Radar Loop"
+description: "Instructions for integrating bomradarloop within Home Assistant"
+date: 2019-02-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Camera
+logo: home-assistant.png
+ha_release: 0.88
+ha_iot_class: "depends"
+---
+
+The `bomraradarloop` platform uses the Australian Bureau of Meteorology (BOM) radar web service as a source to generate an animated radar image using the camera component.
+
+## {% linkable_title Configuration %}
+
+To enable `bomradarloop` in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: bomradarloop
+    location: yourlocation
```

Comment:
```suggestion
    location: YOUR_LOCATION
```

Reply:
Fixed, thank you -- that was in the instructions and I overlooked it.